### DISCUSSION TITLE
Which CI tool to use?

### DISCUSSION THREAD

**Discussion by pfazekas1** [pfazekas1/stellar_dew#17] [General]
Hey. So the question is simple. Do anyone has any experience with CI? Also, I think we should use CircleCI because it can work with github, and it is free for the applications we will use it for. At least thats what I read. 

Please let me know what you think about this.

**Comment by TimiBodi** [comment]
Hello. I don,t have too much experience with it. We talked about it a little in person but since it is a good idea for the others to see it to I will write it here as well. I think you have the most experience out of all of us so we will believe your judgment. I also looked it up a bit and from what i understood I also think it is a good idea to use CirlcleCI
